analysers/analyser_merge_power_substation_minor_FR.py

```diff
@@ -48,19 +48,19 @@ def __init__(self, config, logger = None):
                 select = Select(
                     types = ["nodes", "ways"],
                     tags = [
-                        {"power": "substation", "substation": "minor_distribution"},
-                        {"power": None, "transformer": ["distribution", "main"]},
-                        {"power": "substation", "substation": "distribution"}]),
+                        {"power": "substation", "substation": ["distribution", "minor_distribution"]},
+                        {"power": None, "transformer": ["distribution", "main"]}]),
                 conflationDistance = 50,
                 mapping = Mapping(
                     static1 = {
                         "power": "substation",
                         "voltage": "20000"},
                     static2 = {
-                        "substation": "minor_distribution"},
-                    mapping2 = {
+                        "substation": "minor_distribution"}, # Currently default value, we're unable to destinguish distribution and minor_distribution in opendata
+                    mapping1 = {
                         "operator": "GRD",
```

Comment:
Adding tags to static1/maping1 will raise again Osmose issues for all objects without thus tags.
It is already the largest item of Osmose... I am not in favour of re-raising  issue on existing substation.

Reply:
Indeed